specification/maps/data-plane/Creator/preview/2022-09-01-preview/wayfind.json

```diff
@@ -0,0 +1,308 @@
+        },
+        "lengthInMeters": {
+          "type": "number",
+          "description": "The leg length, in meters, of this leg."
+        },
+        "timeInSeconds": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The leg travel time, in seconds, to travel between the first and last point of this leg."
+        },
+        "startLevel": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The floor where this leg starts."
+        },
+        "endLevel": {
+          "type": "integer",
+          "format": "int32",
+          "description": "The floor where this leg ends."
+        },
+        "points": {
+          "description": "The leg shape points. Their vertical position is determined as follows: all the points are at the same height when startLevel and endLevel are the same, otherwise the array will contain only two points: the first at startLevel and the second at endLevel.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Point"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "Path": {
+      "description": "The details about a particular path between the origin and destination. It can be made up of one or more legs.",
+      "type": "object",
+      "properties": {
+        "lengthInMeters": {
+          "type": "number",
+          "description": "The path length, in meters, of the entire path."
+        },
+        "timeInSeconds": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The path total time, in seconds, to travel between the origin and the destination when following this path."
+        },
+        "legs": {
+          "description": "The different travel sections of this path.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Leg"
+          },
+          "x-ms-identifiers": []
+        }
+      }
+    },
+    "WayfindResult": {
+      "description": "The object returned by this Wayfinding Path request.",
+      "type": "object",
+      "properties": {
+        "noResultExplanation": {
+          "type": "string",
+          "description": "In the event that no paths are found, the `paths` array will be empty. If the reason no paths were found can be determined, this property will provide an explanation why.",
+          "enum": [
+            "NoExplanation"
+          ],
+          "x-ms_enum": {
+            "name": "noResultExplanation",
+            "description": "In the event that no paths are found, the `paths` array will be empty. If the reason no paths were found can be determined, this property will provide an explanation why.",
+            "values": [
+              "NoExplanation"
+            ],
+            "modelAsString": true
+          }
```

Comment:
Given modelAsString=true, this could have more value than "NoExplanation". But do we have other code than this "NoExplanation" code to be documented?

Reply:
initially it was going to be just a string, but the review board suggested using an extensible enum.  for now it can only have a value and it will expand in the future as we learn more about the behavior.